src/gui/tray/CustomButton.qml

```diff
@@ -0,0 +1,68 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
+
+Button {
+    id: root
+
+    property string imageSource: ""
+    property string imageSourceHover: ""
+
+    property string toolTipText: ""
+
+    property color textColor: ({})
+    property color textColorHovered: ({})
+
+    property color bgColor: "transparent"
+
+    property bool bold: false
+
+    Accessible.role: Accessible.Button
+    Accessible.name: text !== "" ? text : (toolTipText !== "" ? toolTipText : qsTr("Activity action button"))
+    Accessible.onPressAction: clicked()
+
+    background: Rectangle {
+        anchors.fill: parent
+        color: root.bgColor
+        opacity: parent.hovered ? 1.0 : 0.3
+        radius: 25
+    }
+
+    contentItem: RowLayout {
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.leftMargin: root.text === "" ? 5 : 10
```

Comment:
Button inherit from QQC2.Control, so you should be using padding/inset instead of anchoring the element :) https://doc.qt.io/qt-5/qml-qtquick-controls2-control.html#control-layout

Reply:
@CarlSchwan Not sure I get it, as, inside the `RowLayout` I can't use mentioned properties (should I wrap it into something?). But, I will take my time to read that doc.

Reply:
so what @CarlSchwan means is that you could set the size and position in the following way
```
RowLayout {
    x: 0
    y: 0
    width: root.width
    height: root.height
    leftPadding: root.text === "" ? 5 : 10
    rightPadding: root.text === "" ? 5 : 10
```

Reply:
Actually it is more  like this:

```
Button {
        leftPadding: root.text === "" ? 5 : 10
        rightPadding: root.text === "" ? 5 : 10
        contentItem: RowLayout {
```

No need to specifiy x, y, width and height for the RowLayout

Reply:
thanks @CarlSchwan

Reply:
@CarlSchwan @mgallien These `padding` values have no effect in my case. See the screenshot (I've set them to 50 already and this doesn't work)
![image](https://user-images.githubusercontent.com/12224014/151165430-e9ae2edf-7085-4470-9333-bab48c0bac4c.png)

Reply:
> so what @CarlSchwan means is that you could set the size and position in the following way
> 
> ```
> RowLayout {
>     x: 0
>     y: 0
>     width: root.width
>     height: root.height
>     leftPadding: root.text === "" ? 5 : 10
>     rightPadding: root.text === "" ? 5 : 10
> ```

@mgallien QML is complaining `Invalid property name "leftPadding` and `Invalid property name "rightPadding"&#129335;

Reply:
@CarlSchwan @mgallien can one of you add a commit with a correct implementation that is confirmed to be working in a way that it looks the same as on the latest GIF approved by Jan?

Reply:
but you do not need any anchor between what you use for the  `contentItem` item and the parent control
this comes for free and is the reason why padding is getting ignored

Reply:
example
```
Button {
    id: root

    leftPadding: 50
    rightPadding: 50

    contentItem: RowLayout {
        Image {
            id: icon

            Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter

            source: root.hovered ? root.imageSourceHover : root.imageSource
        }
```

Reply:
@mgallien Indeed. That worked. Thx.